packages/plugin-logger/src/main.js

```diff
@@ -257,6 +259,10 @@ let $registerImaPlugin = ns => {
   ns.namespace('plugin.logger');
 };
 
+pluginLoader.register('@ima/plugin-logger', ns => {
+  $registerImaPlugin(ns);
```

Comment:
On other note, not sure if the ns definition is of any use.... but probably should be inlined, we don't want to have any references to `$registerImaPlugin` function in future versions as it is legacy and unused API.